main_test.go

```diff
@@ -22,7 +22,8 @@ import (
 	"testing"
 )
 
-func TestMain(t *testing.T) {
+
+func Test_Main(t *testing.T) {
```

Comment:
@wayilau thank you for the PR but I didn't understand the motivation behind the changes. Please reopen with more information if needed. Thanks!

Reply:
Because i want run tests in IDE, but it can not be executed. so i changed the method name,  it works. @spzala

Reply:
@wayilau hmmm...OK, I will keep it open for review, can you please add more description, any error that you are getting etc. Thanks!

Reply:
it tells me that it is a wrong test signature, so i can not run it in the ide. after i change it to Test_Main, it works. As TestMain is a test function used for testing.M, so we can use  the 'Run' Method to tests our  code,  the goal is  init or clean resourcds,  but  here we are not. so i changed it  to Test_Main. @spzala